csrc/executor.cpp

```diff
@@ -331,6 +331,29 @@ void FusionExecutor::compileFusion(
   lowered_->run();
 
   kir::Kernel* kernel = lowered_->kernel();
+
+  const kir::KernelSummary& kernel_summary = kernel->summary();
+
+  // TODO: this replicates the target GPU version computation from
+  // executor_utils.
+  std::pair<int, int> target_arch;
+  bool compile_to_sass = false;
+  executor_utils::queryTargetGPUVersion(
+      properties, target_arch.first, target_arch.second, compile_to_sass);
```

Comment:
```suggestion
  executor_utils::queryTargetGPUVersion(
      properties, std::ref(target_arch.first), std::ref(target_arch.second), /*compile_to_sass=*/false);
```

std::ref makes it clear to the reader that this variable can be modified by the callee.

Reply:
I don't have much experience with `std::ref` but that sounds good to me.